src/proteomes/components/entry/ComponentsDownload.tsx

```diff
@@ -106,19 +108,20 @@ const ComponentsDownload = ({
     namespace,
   };
 
-  const isoformsAvailable = Boolean(statistics.isoforms);
+  const isoformsAvailable = Boolean(proteomeStatistics.isoformProteinCount);
   const nSelectedEntries = numberSelectedEntries || selectedEntries.length;
   let downloadCount;
   switch (downloadSelect) {
     case 'all':
       downloadCount = includeIsoform
-        ? totalNumberResults + (statistics.isoforms || 0)
+        ? totalNumberResults + (proteomeStatistics.isoformProteinCount || 0)
         : totalNumberResults;
       break;
     case 'reviewed':
       downloadCount = includeIsoform
-        ? (statistics?.reviewed || 0) + (statistics.isoforms || 0)
-        : statistics?.reviewed || 0;
+        ? (proteomeStatistics.reviewedProteinCount || 0) +
+          (proteomeStatistics.isoformProteinCount || 0)
+        : proteomeStatistics.reviewedProteinCount || 0;
```

Comment:
Suggesting only in the case that you like it but otherwise what is already there is completely fine:
```
      downloadCount =
        ((includeIsoform && proteomeStatistics.isoformProteinCount) || 0) +
        (proteomeStatistics.reviewedProteinCount || 0);
```